DeploymentExperiences.md

```diff
@@ -0,0 +1,5 @@
+## Deployment Experiences
+
+Deploying my Djorg Application proved to be a difficult task today- A task that as of now is not complete.
+At the current time, my Djorg Application is deployed and can be reached by using the link provided in the Links.md file. However, while the Application is deployed, clicking the bookmarks link from the homepage results in a Server Error(500). 
+The process of deploying this application was tricky. I started the process going through the high-level steps provided in the README. I was exicted when I finally "Deployed" by entering `git push heroku master` into the command line. Quickly this excitment faded once i went to the page and was presented with an Application Error- that told me to check the logs.  I checked the logs using 'heroku logs --tail --app sprintdjorg' and quickly spotted a bunch of thing i had done wrong. Slowly- I made the changes that I thought would fix my errors. I finally got my deployment to the state it is in now- deploying but it not working. I then tried changing random things that i thought would lead to a successful deployment- after over an hour of "Application Error" and the Application deploying but not working properly I decided to move on. Hopefully. after finishing the remaining tasks of todays Sprint Challenge, I will be able to continue working until have a working deployment. If I run out of time - Then It is my goal to learn what I am doing wrong- in todays Solution Lecture.
```

Comment:
Awesome! Deployment can be super tricky but I'm really glad that you were able to work through the errors and get deployment finished. Being able to solve problems like this is a really important skill for us as developers.